versions.json

```diff
@@ -100,88 +100,88 @@
             },
             "fabricmanager": {
                 "distribution": "rhel8",
-                "version": "550.90.07-1",
+                "version": "550.90.07",
                 "sha256": "e580c8b412de7a9f352c2eacce877b4cbccd494bfdee7a0e6c94010a06c91c92"        
             }   
         }
     },
     "cuda": {
         "ubuntu20.04": {
             "driver": {
-                "version": "12-4",
+                "version": "12.5",
                 "distribution": "ubuntu2004"
             },
             "samples": {
-                "version": "12.4",
-                "sha256": "aa28fa2227768dd31ebbf9cd48b265a0c8810fae03e02c6079c0fa71bbea7319"
+                "version": "12.5",
```

Comment:
Would it be possible to move samples under a common distribution?

Reply:
As it's structured at the moment, no, as common only works at the top level. I'd need to pull cuda-driver and cuda-samples out separately (which might make sense).

The other option is to pull `distribution` out of the config (as it doesn't change release to release) and put it in `set_properties.sh`. That would allow all of `cuda` to be common. But if I did that I'd probably like to apply that pattern a few places and would probably be better suited for a new PR.